stories/custom-dialog/digit-custom-dialog.stories.mdx

```diff
@@ -16,11 +16,11 @@ Used when you need more customization then `useDigitDialog`. You're able to prov
     <Story name="dialog">
         {() => {
             const [openDialog] = useDigitCustomDialog({
-                title: "Oh"
+                renderTitle: () => <button> with a button? </button>
```

Comment:
`renderTitle` example should be a new `useDigitCustomDialog` imo. But there's larger problem with this demo that is outside the scope of this PR, so I don't really mind if we let this one slip. The larger problem being not enough examples demonstrating the power of `useDigitCustomDialog`.